eva-accession-import-automation/create_fasta_from_assembly_report.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# Usage: create_fasta_from_assembly_report.sh </path/to/assembly_report.txt>
+
+if [ "$#" -lt 3 ]
+then
+    echo "Please provide the assembly accession, the path to the assembly report and the output folder."
+    exit 1
+fi
+
+assembly_accession=$1
+assembly_report=$2
+output_folder=$3
+
+for i in `grep -v -e "^#" ${assembly_report} | cut -f5`;
```

Comment:
replace `i` for `genbank_contig`? if you do this, do it after fixing the other comments